tests/support/Pipelines/Bbh/Test_FindHorizon.py

```diff
@@ -0,0 +1,42 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import logging
+import os
+import unittest
+
+from spectre.Informer import unit_test_src_path
+from spectre.Pipelines.Bbh.FindHorizon import find_horizon
+from spectre.Visualization.OpenVolfiles import open_volfiles
+from spectre.Visualization.ReadH5 import list_observations
+
+
+class TestFindHorizon(unittest.TestCase):
+    def setUp(self):
+        self.h5_filename = os.path.join(
+            unit_test_src_path(), "Visualization/Python", "VolTestData0.h5"
+        )
+
+    def test_find_horizon(self):
+        obs_ids, obs_times = list_observations(
+            open_volfiles([self.h5_filename], "/element_data")
+        )
+        # Can't test more than that this code runs until it errors until we have
+        # a way to generate data with an apparent horizon.
+        with self.assertRaisesRegex(
+            RuntimeError, "Failed to open dataset 'InverseSpatialMetric_xx'"
+        ):
+            find_horizon(
```

Comment:
For some reason gcc isn't liking this.

Reply:
tried another way of calling this

Reply:
didn't work. I just disabled this test for gcc (see comment in cmake)